tests/hazmat/backends/test_openssl.py

```diff
@@ -78,7 +75,7 @@ def test_supports_cipher(self):
         assert backend.cipher_supported(None, None) is False
 
     def test_aes_ctr_always_available(self):
```

Comment:
This test was only to verify our 0.9.8/1.0.0 trick of supporting CTR was present. It can be entirely removed now.